src/devices/Bmp180/Bmp180.cs

```diff
@@ -105,11 +116,11 @@ public double ReadAltitude(double seaLevelPressure = 101325.0)
         ///  altitude in meters
         /// </param>
         /// <returns>
-        ///  Pressure in Pascals
+        ///  Pressure
         /// </returns>
-        public double ReadSeaLevelPressure(double altitude = 0.0)
+        public Pressure ReadSeaLevelPressure(double altitude = 0.0)

```

Comment:
We should consider moving this to WeatherHelpers

Reply:
Yes definitely, I will move them but this will be in a completely different branch. I don't want to mix the two up.